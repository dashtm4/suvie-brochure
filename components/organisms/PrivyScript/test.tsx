import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { PrivyScript } from '.'

describe('<PrivyScript />', () => {
  it('should renders correct <PrivyScript />', () => {
    const wrapper = shallow(<PrivyScript />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
