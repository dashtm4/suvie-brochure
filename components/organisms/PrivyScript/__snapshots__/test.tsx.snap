// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PrivyScript /> should renders correct <PrivyScript /> 1`] = `
<Fragment>
  <script
    dangerouslySetInnerHTML={
      Object {
        "__html": "var _d_site = _d_site || '61530AE1C7663076284C9C9F';",
      }
    }
  />
  <script
    src="https://widget.privy.com/assets/widget.js"
  />
</Fragment>
`;
